geomstats/_backend/__init__.py

```diff
@@ -111,7 +111,7 @@
         'zeros',
         'zeros_like'
     ],
-    'autograd': ['value_and_grad'],
+    'autograd': ['jacobian', 'value_and_grad'],
```

Comment:
You have to add a `jacobian` function to the other backends, in the [autograd](https://github.com/geomstats/geomstats/blob/master/geomstats/_backend/pytorch/autograd.py) module.

Reply:
I'm not sure I get it: do you mean that I should write a function that does the same thing as ```jacobian``` of autograd but for the other backends ?

Reply:
Sorry I didn't see this earlier